CHANGELOG.md

```diff
@@ -6,9 +6,11 @@ _The format is based on [Keep a Changelog](http://keepachangelog.com/) and this
 
 ## Unreleased
 
+- Implement `shouldForwardProp` API for native and primitive platforms (see [#3093](https://github.com/styled-components/styled-components/pull/3107))
+
 ### Bugfixes
 
-- Added `useTheme` hook to named exports for react-primitives entrypoint (see [#2982](https://github.com/styled-components/styled-components/pull/2982)) thanks @jladuval!
+- Added `useTheme` hook to named exports for react-primitives entrypoint (see [#2982](https://github.com/styled-components/styled-components/pull/3108))
```

Comment:
I think this might have been a mistake?

Reply:
There were two PRs for this, a previous one exporting `useTheme` from the native entrypoint done by someone else, but I created one recently exporting `useTheme` in the primitives entrypoint, but referenced the original native PR in the changelog.

Someone added a thanks note in the `CHANGELOG.md` which is in `master` now, referencing the author of the native PR, because I had linked to the wrong PR, since I hadn't opened the one for the primitives entrypoint yet. Was correcting the PR reference here to the one that I opened for `primitives`, since the `native` one has already been published and is already in the `CHANGELOG.md` for an earlier release.

I hope that makes sense 😄.

Reply:
Oh ok, can you fix the link text to match the PR that was linked?

Reply:
`2982` has already been published, it should've been mentioned here: https://github.com/styled-components/styled-components/blob/88ec191fdef0e0a07b40190170d60b01dee261a1/CHANGELOG.md#v501---2020-02-04

`3108` is the correct PR for adding `useTheme` to the `react-primitives` entrypoint.

Reply:
Are you asking me to revert it back to `Added ``useTheme`` hook to named exports for react-primitives entrypoint (see [#2982](https://github.com/styled-components/styled-components/pull/2982)) thanks @jladuval!`?

I can do so, but `2982` is for the native PR though. `3108` is for my `react-primitives entrypoint` PR, which this changelog entry is referring to.

This is the commit where the wrong thanks note (for native, not primitives) was added: https://github.com/styled-components/styled-components/commit/01e5747aa4be056cbc2712e53a588b2ac7157d41#diff-4ac32a78649ca5bdd8e0ba38b7006a1eL9

Sorry for the confusion! I'm a bit new to changelogs. Am happy to change back to your suggestion, am trying to avoid confusion about the wrong PR link in the changelog though.